cekit-helpers/cekit-cache-server/src/main/java/org/kie/cekit/cacher/builds/nightly/NightlyBuildsPullRequestAcceptor.java

```diff
@@ -131,12 +110,12 @@ public void onFilePersisted(String fileName, String checkSum) {
 
                                 log.fine(String.format("Updating BC monitoring rhpam_business_central_monitoring_distribution.zip from [%s] to [%s]", artifact.getMd5(), bcMonitoringCheckSum));
                                 artifact.setMd5(bcMonitoringCheckSum);
-                                yamlFilesHelper.writeModule(bcMonitoring, bcMonitoringFile);
+                                yamlFilesHelper.writeModule( buildUtils.bcMonitoring(),  buildUtils.bcMonitoringFile());
```

Comment:
formatting change: removing white spaces
```suggestion
                                yamlFilesHelper.writeModule(buildUtils.bcMonitoring(), buildUtils.bcMonitoringFile());
```